README.md

```diff
@@ -1,15 +1,111 @@
+- **dbt Version**: This dbt package requires you have a functional dbt project that utilizes a dbt version within the respective range `>=1.0.0, <2.0.0`.
+## Step 2: Installing the Package
+Include the following apple_store_source package version in your `packages.yml`
+> Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+```yaml
+packages:
+  - package: fivetran/apple_store_source
+    version: [">=0.1.0", "<0.2.0"]
+```
+## Step 3: Configure Your Variables
+### Database and Schema Variables
+By default, this package will run using your target database, the `itunes_connect` schema and the default Fivetran Source Tables names for Apple App Store listed above. If this is not where your Apple App Store data is (perhaps your Apple App Store schema is `itunes_connect_fivetran` your `app` table is named `usa_app`), add the following configuration to your `dbt_project.yml` file:
 
-Try running the following commands:
-- dbt run
-- dbt test
+```yml
+vars:
+  apple_store_source:
+    apple_store_database: your_database_name
+    apple_store_schema: your_schema_name 
+    <source_table_name>_identifier: your_table_name
+```
+### Enabling Components
+Your Apple App Store connector might not sync every table that this package expects. If you use subscriptions and have the `sales_subscription_event_summary` and `sales_subscription_summary` tables synced, add the following variable to your `dbt_project.yml` file:
 
+```yml
+vars:
+  apple_store__using_subscriptions: true # by default this is assumed to be false
+```
 
-### Resources:
-- Learn more about dbt [in the docs](https://docs.getdbt.com/docs/introduction)
-- Check out [Discourse](https://discourse.getdbt.com/) for commonly asked questions and answers
-- Join the [chat](https://community.getdbt.com/) on Slack for live discussions and support
-- Find [dbt events](https://events.getdbt.com) near you
-- Check out [the blog](https://blog.getdbt.com/) for the latest news on dbt's development and best practices
+Additionally, by default, `Subscribe`, `Renew` and `Cancel` subscription events are included and required in this package for downstream usage. If you would like to add additional subscription events, please add the below to your `dbt_project.yml`:
+
+```yml
+    apple_store__subscription_events:
+    - 'Renew'
+    - 'Cancel'
+    - 'Subscribe'
+    - '<additional_event_name>'
+    - '<additional_event_name>'
+```
+
+## (Optional) Step 4: Additional Configurations
+### Change the Build Schema
+By default, this package builds the Apple App Store staging models within a schema titled (<target_schema> + `_stg_apple_store`) in your target database. If this is not where you would like your Apple App Store staging data to be written to, add the following configuration to your `dbt_project.yml` file:
```

Comment:
`_stg_apple_store` or `_apple_store_source`?

Reply:
I have `stg_apple_store` in the `dbt_project.yml` right now! So this should be correct - but adding onto convo sync for us later